client/js/components/addon/AddonWrapper.vue

```diff
@@ -75,7 +76,11 @@ export default {
             this.$options.components[addon.entry] = window[addon.entry].default
 
             this.component = window[addon.entry].default
+
+            this.isLoaded = true
           }
+
+          this.$emit('loaded', this.isLoaded)
```

Comment:
As i understand it, if `addons.assets.load` results in no response (aka there is no addon installed which uses the hook), the payload of the 'loaded' event will be `false`, which can be used by the parent component to apply its logic. this is a sufficient solution imho.

Reply:
Yes, that was the idea. But @salisdemos mentioned that we can load more than one addon per hook, so I changed it. Now the event emits an array with all loaded addons and in the parent you can check if a specific addon was loaded (with the entry name). Done in [b0c4de4](https://github.com/demos-europe/demosplan-core/pull/2816/commits/b0c4de4518d4aa4995bbdbccae217fc67ed23a90)